cloud/deployment/deployment.go

```diff
@@ -274,105 +291,275 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeplo
+		}
+		if IsDeploymentStandard(deploymentType) {
+			var requestedCloudProvider astroplatformcore.CreateStandardDeploymentRequestCloudProvider
+			if cloudProvider == gcpCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderGCP
+			}
+			if cloudProvider == awsCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderAWS
+			}
+			if cloudProvider == azureCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderAZURE
+			}
+			var requestedExecutor astroplatformcore.CreateStandardDeploymentRequestExecutor
+			if executor == CeleryExecutor {
+				requestedExecutor = astroplatformcore.CreateStandardDeploymentRequestExecutorCELERY
+			}
+			if executor == KubeExecutor {
+				requestedExecutor = astroplatformcore.CreateStandardDeploymentRequestExecutorKUBERNETES
+			}
+			standardDeploymentRequest := astroplatformcore.CreateStandardDeploymentRequest{
+				AstroRuntimeVersion:  runtimeVersion,
+				CloudProvider:        &requestedCloudProvider,
+				Description:          &description,
+				Name:                 name,
+				Executor:             requestedExecutor,
+				IsCicdEnforced:       isCicdEnforced,
+				Region:               &region,
+				IsDagDeployEnabled:   dagDeployEnabled,
+				IsHighAvailability:   highAvailabilityValue,
+				WorkspaceId:          workspaceID,
+				Type:                 astroplatformcore.CreateStandardDeploymentRequestTypeSTANDARD,
+				DefaultTaskPodCpu:    defaultTaskPodCpu,
+				DefaultTaskPodMemory: defaultTaskPodMemory,
+				ResourceQuotaCpu:     resourceQuotaCpu,
+				ResourceQuotaMemory:  resourceQuotaMemory,
+			}
+			if executor == CeleryExecutor {
+				standardDeploymentRequest.WorkerQueues = &defautWorkerQueue
+			}
+			if schedulerSize != "" {
+				if schedulerSize == SmallScheduler {
+					standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSizeSMALL
+				}
+				if schedulerSize == MediumScheduler {
+					standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSizeMEDIUM
+				}
+				if schedulerSize == LargeScheduler {
+					standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSizeLARGE
+				}
+			} else {
+				standardDeploymentRequest.SchedulerSize = astroplatformcore.CreateStandardDeploymentRequestSchedulerSize(configOption.DefaultValues.SchedulerSize)
+			}
+			if highAvailability == enable { //nolint: goconst
+				standardDeploymentRequest.IsHighAvailability = true
+			}
+			err := createDeploymentRequest.FromCreateStandardDeploymentRequest(standardDeploymentRequest)
+			if err != nil {
+				return err
+			}
+		}
+		// build dedicated input
+		if IsDeploymentDedicated(deploymentType) {
+			var requestedExecutor astroplatformcore.CreateDedicatedDeploymentRequestExecutor
+			if executor == CeleryExecutor {
```

Comment:
pretty much same comments from above

Reply:
what do you mean this is where the dedicated input is built?